inductiva/resources/storage.py

```diff
@@ -1,15 +1,89 @@
 """Methods to interact with the user storage resources."""
 import inductiva
 from inductiva.client.apis.tags import instance_api
+from inductiva.utils import format_utils
+from typing import Literal
 
 
 def get_space_used():
     """Returns the occupied storage size in GB."""
     try:
         api = instance_api.InstanceApi(inductiva.api.get_client())
-        instance_price = api.get_storage_size()
-        instance_price = float(round(instance_price.body, 3))
-        print(f"Total storage used: {instance_price} GB")
-        return instance_price
+        response = api.get_storage_size()
+        storage_used = round(float(response.body), 3)
+        print(f"Total storage used: {storage_used} GB")
+        return storage_used
     except inductiva.client.ApiException as api_exception:
         raise api_exception
+
+# pylint: disable=redefined-builtin
```

Comment:
A couple of questions: 
1. I think we already had a couple of methods called list in the past? @begaiym-k did @luispcunha we also disabled pylint?
2. If so, does it make sense? Or should we rename the methods to something slightly different? For storage, a natural choice would be `ls`... But not sure for tasks and machine groups, wether it would feel natural...

Any suggestions?

Reply:
Additionally, I think we need to think a bit better about the names of the methods.
For example, should we follow this pattern:

```
https://www.geeksforgeeks.org/python-list-files-in-a-directory/
```

I would prefer to follow existing patterns.
Would:
```
inductiva.os.listdir()
```

make more sense?

Reply:
Should we change the location of the storage functions and create a module called storage like 
`inductiva.storage.list()`

Reply:
I think I prefer the inductiva.os and then use the python os package patterns.
@hpenedones wdyt?

Reply:
What will os include @sarmento? Just the storage or other resources as well?

Reply:
Just adding to what @luispcunha said earlier, `os` as a library allows access the file directory, change the file locations and rename them, that's intuitive for the user when dealing with os. however, we are not providing any of those features and I don't think it makes sense to do it from here, especially if we want to build the webpage for that purpose.

Reply:
I think os could have things related with environment variables, the API keys, user info, etc.
For example, I could see these workin

```
os.chdir(path)
os.fchdir(fd)
os.getcwd()
```

but even if we use
``` 
inductiva.storage
```
I suggest we stick to names of function that are the same as python os functions. For exemple:

```
inductiva.storage.listdir()
```

let's stick to standards as much as possible (assuming that funcionalities are the same/similar).

Reply:
@luispcunha : what about inductiva.storage.listdir()?

Reply:
I think @sarmento suggestion here could be neat:

```python 
inductiva.storage.listdir()
```

instead of 

```python
inductiva.resources.storage.list_contents()
```

as it is more compact, and it is also a familiar method name. We can see later down the line if we start implementing a lot of `os` commands and would benefit from a packing rename. I hope we don't need to implement many of those low-level things, and users can use either pyarrow or other libraries directly, but let's see...

Regarding the other places where we over-wrote the Python keyword `list`, I would see two options:

1) we either also find a natural name alternative that is both compact and standard (but might be hard).
2) or perhaps we implement @luispcunha suggestion with the `__init__.py` trick....

Reply:
This chat is near real-time, when I posted my answer as already outdated by two other comments! LOL